### PR TITLE
[wg](chore) rename workload group memory property names

### PR DESCRIPTION
**Pull request by jacktengg**
### What problem does this PR solve?

This is a property behaviour change. Since spill disk is not many used, we can change its name.

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [x] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label 

Mannual test steps:
1. With old FE and BE, modify properties to new value:
```
alter workload group normal properties("spill_threshold_low_watermark"="55%");
alter workload group normal properties("spill_threshold_high_watermark"="85%");
```
2. Update BE to the modified code, check that values are the update values in step 1.
3. Modify properties values again:
```
alter workload group normal properties("spill_threshold_low_watermark"="60%");
alter workload group normal properties("spill_threshold_high_watermark"="90%");
```
4. Update FE to the modified code,  check that values are the update values in step 3.
5. Modify properties values again:
```
alter workload group normal properties("memory_low_watermark"="65%"); 
alter workload group normal properties("memory_high_watermark"="95%"); 
```

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]

File: gensrc/thrift/BackendService.thrift [hunk 1]
```diff
@@ -265,8 +265,8 @@ struct TWorkloadGroupInfo {
   9: optional i32 scan_thread_num
   10: optional i32 max_remote_scan_thread_num
   11: optional i32 min_remote_scan_thread_num
-  12: optional i32 spill_threshold_low_watermark
-  13: optional i32 spill_threshold_high_watermark
+  12: optional i32 memory_low_watermark
+  13: optional i32 memory_high_watermark
```
Comment (wangbo) [inline gensrc/thrift/BackendService.thrift:269]: Did a upgrade test here? Such as  a old FE and a new BE with this PR, what could happen?

File: be/src/runtime/workload_group/workload_group.h [hunk 2]
```diff
@@ -95,10 +95,10 @@ class WorkloadGroup : public std::enable_shared_from_this<WorkloadGroup> {
     void do_sweep();
 
     int spill_threshold_low_water_mark() const {
```
Comment (wangbo) [inline be/src/runtime/workload_group/workload_group.h:97]: it should be renamed.

File: be/src/runtime/workload_group/workload_group.h [hunk 3]
```diff
@@ -95,10 +95,10 @@ class WorkloadGroup : public std::enable_shared_from_this<WorkloadGroup> {
...
-        return _spill_low_watermark.load(std::memory_order_relaxed);
+        return _memory_low_watermark.load(std::memory_order_relaxed);
     }
     int spill_threashold_high_water_mark() const {
```
Comment (wangbo) [inline be/src/runtime/workload_group/workload_group.h:100]: it should be renamed.

File: gensrc/thrift/BackendService.thrift (hunk 1 above)
Comment (jacktengg) [reply gensrc/thrift/BackendService.thrift:269]: Test is done and it's OK to rename a field.

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**